traffic_ops/traffic_ops_golang/api/api.go

```diff
@@ -1075,6 +1102,34 @@ func GetUserFromReq(w http.ResponseWriter, r *http.Request, secret string) (auth
 	return user, nil, nil, http.StatusOK
 }
 
+func getCookieFromAccessToken(bearerToken string, secret string) (*http.Cookie, error) {
+	var cookie *http.Cookie
+	claims := jwt.MapClaims{}
+	token, err := jwt.ParseWithClaims(bearerToken, claims, func(token *jwt.Token) (interface{}, error) {
+		return []byte(secret), nil
+	})
+	if err != nil {
+		return nil, fmt.Errorf("parsing claims: %w", err)
+	}
+	if !token.Valid {
```

Comment:
I didn't look in the underlying functions, but can there be a case where there is no error, but the returned token pointer is `nil`? 
If so, this will panic in that case.
If not, then disregard this comment :)

Reply:
not that i can find but no reason not to check! ill put it in there